reset.py

```diff
@@ -157,6 +158,14 @@ def transition_reset(self):
                 values=['cancelled', '%s,%s' % (Reset.__name__, reset.id)],
                 where=sql_where))
 
+            products = set([m.product for m in self.confirm.moves])
+            date = min([m.effective_date or today for m in self.confirm.moves]
+                + [production.effective_date or today])
+
+            with Transaction().set_context(queue_name='production'):
```

Comment:
It's better not to use a queue_name by default. That's what's been done in core and allows for better customization.

Reply:
See that production_quality_control module is already using a queue by "production" name

Reply:
287dcda50c5c01b8ce7cc89b833f784734469f23